MSAL/src/native_auth/network/responses/validator/token/validated_response/MSALNativeAuthTokenValidatedResponse.swift

```diff
@@ -108,6 +108,7 @@ enum MSALNativeAuthTokenValidatedErrorType: Error {
         }
     }
 
+    // swiftlint:disable:next function_body_length
```

Comment:
do we need this new line if we didn't modify this function?

Reply:
Removed